src/main/java/edu/unc/lib/boxc/migration/cdm/services/ArchivalDestinationsService.java

```diff
@@ -43,6 +44,9 @@ public class ArchivalDestinationsService {
     private String solrServerUrl;
     private HttpSolrClient solr;
 
+    public static final String PID = SearchFieldKey.ID.getSolrField();
```

Comment:
I'd suggest not naming the constant this, boxc has a PID class that is used all over the place which could conflict or at least caused confusion. Naming it "PID_KEY" would would fine, or "ID_KEY"